packages/entity/src/EntityFields.ts

```diff
@@ -1,16 +1,110 @@
+import { IEntityClass } from './Entity';
+import EntityPrivacyPolicy from './EntityPrivacyPolicy';
+import ReadonlyEntity from './ReadonlyEntity';
+import ViewerContext from './ViewerContext';
+
+export enum EntityEdgeDeletionBehavior {
+  /**
+   * Default. Invalidate the cache for all entities that reference the entity
+   * being deleted through this field. This is most useful when the database itself expresses foreign
```

Comment:
Talk about how this also runs the respective edge deletion policies for all transitive inbound edges.